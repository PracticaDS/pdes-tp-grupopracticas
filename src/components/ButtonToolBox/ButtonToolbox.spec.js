import React from 'react';
import ReactDOM from 'react-dom';
import ButtonToooBox from './ButtonToolBox'

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ButtonToooBox />, div);
  ReactDOM.unmountComponentAtNode(div);
});