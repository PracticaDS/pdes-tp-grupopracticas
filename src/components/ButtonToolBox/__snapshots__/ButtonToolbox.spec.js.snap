// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonToolBox should render correctly whith props 1`] = `
<div
  className="ButtonToolbox"
  onClick={[Function]}
>
  <img
    alt="actions"
    src="src_img"
  />
</div>
`;

exports[`ButtonToolBox should render correctly whithout props 1`] = `
<div
  className="ButtonToolbox"
  onClick={[Function]}
>
  <img
    alt="actions"
  />
</div>
`;
